pkg/proxy/apis/config/types.go

```diff
@@ -307,33 +303,3 @@ func (m *LocalMode) String() string {
 func (m *LocalMode) Type() string {
 	return "LocalMode"
 }
-
-type ConfigurationMap map[string]string
```

Comment:
dupe of https://github.com/kubernetes/kubernetes/blob/ed84e7f6d327d518e140c5f9c31462075b922c3a/staging/src/k8s.io/component-base/cli/flag/configuration_map.go#L25-L52